### PR TITLE
dotnet pack returns the wrong exit code

### PR DESCRIPTION
**Pull request by victorhurdugaci**
Fixes https://github.com/dotnet/cli/issues/333

Because true is false

Please review @anurse @davidfowl

### CODE REVIEW

File: src/Microsoft.DotNet.Tools.Pack/Program.cs [hunk 1]
```diff
@@ -51,7 +51,7 @@ public static int Main(string[] args)
                 var configValue = configuration.Value() ?? Cli.Utils.Constants.DefaultConfiguration;
                 var outputValue = output.Value();
 
-                return BuildPackage(path, configValue, outputValue, intermediateOutput.Value()) ? 1 : 0;
+                return BuildPackage(path, configValue, outputValue, intermediateOutput.Value()) ? 0 : 1;
```
Comment (anurse) [inline src/Microsoft.DotNet.Tools.Pack/Program.cs]: Hmm, maybe this should be a helper method?

`Microsoft.Internal.Helpers.ConvertReturnValueFromBooleanToIntegerForProcessExitCode` ? :trollface:

**Comment by anurse** [comment]
:shipit: LGTM

**Comment by Sridhar-MS** [comment]
@dotnet-bot test this please

File: src/Microsoft.DotNet.Tools.Pack/Program.cs (hunk 1 above)
Comment (victorhurdugaci) [inline src/Microsoft.DotNet.Tools.Pack/Program.cs]: const int One = 1;
    const int Zero = 0;

**Pull request merged**